docs/usage.rst

```diff
@@ -59,6 +59,21 @@ RSA encoding and decoding require the ``cryptography`` module. See :ref:`install
     >>> jwt.decode(encoded, public_key, algorithms=["PS256"])
     {'some': 'payload'}
 
+Encoding & Decoding Tokens with EdDSA (Ed25519)
+-------------------------------------------
+
+EdDSA encoding and decoding require the ``cryptography`` module. See :ref:`installation_cryptography`.
+
+.. code-block:: pycon
+    >>> import jwt
+    >>> private_key = "-----BEGIN PRIVATE KEY-----\nMC4CAQAwBQYDK2VwBCIEIPtUxyxlhjOWetjIYmc98dmB2GxpeaMPP64qBhZmG13r\n-----END PRIVATE KEY-----\n"
+    >>> public_key = "-----BEGIN PUBLIC KEY-----\nMCowBQYDK2VwAyEA7p4c1IU6aA65FWn6YZ+Bya5dRbfd4P6d4a6H0u9+gCg=\n-----END PUBLIC KEY-----\n"
+    >>> encoded = jwt.encode({"some": "payload"}, private_key, algorithm="EdDSA")
+    >>> print(encoded)
+    eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCJ9.eyJzb21lIjoicGF5bG9hZCJ9.yPJ7Ggb6aOZn38vNai8bBN6bgfcrzKXSTsdXeGsZGibnxSZrWDrWeB6p35f9RFkMnYgToe_eh4mgye5V6wj5DQ
```

Comment:
```suggestion
```

Reply:
Do we really need to remove `encoded` printing?

If we remove `encoded` printing, I think the `encoded` variable will be unused.

Reply:
the exact string representation of the token matters less and can fail dockets.

`encoded` is used in the decode step below.

Reply:
I see. The `encoded` presenting has been removed.